File: televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/ServerAPI/movie/Scoring.java
package com.vanillax.televisionbingecalculator.app.ServerAPI.movie;

import com.google.gson.annotations.SerializedName;

/**
 * Created by mitchross on 5/20/17.
 */

public class Scoring
{
	@SerializedName( "provider_type" )
	public String providerType;

	@SerializedName( "value" )
	public Double value;
}

File: televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/serverapi/movie/Scoring.kt
package com.vanillax.televisionbingecalculator.app.serverapi.movie

import com.google.gson.annotations.SerializedName

/**
 * Created by mitchross on 5/20/17.
 */

data class Scoring(
        @SerializedName("provider_type")
        val providerType: String? = null,

        @SerializedName("value")
        val value: Double? = null
)
